### PR TITLE
Setting folder

### PR DESCRIPTION
**Pull request by mholmeshackerd**
i did all of this

### CODE REVIEW

File: Scripts/Project_Markdown.Rmd [hunk 1]
```diff
@@ -0,0 +1,949 @@
...
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+---
+
+# Setup 
+```{r setup, include=TRUE}
```
Comment (matt-sasaki) [inline Scripts/Project_Markdown.Rmd:13]: Maybe switch to include = F

File: Scripts/Project_Markdown.Rmd [hunk 2]
```diff
@@ -0,0 +1,949 @@
...
+library(DT)
+library(survival)
+library(ranger)
+library(survminer)
+library(ggfortify)
+library(emmeans)
+#library(plotrix)
+library(MASS)
+
+theme_mat = function(base_size){
```
Comment (rabatts) [inline Scripts/Project_Markdown.Rmd:38]: Keep up the good work

**Comment by matt-sasaki** [comment]
Looks good, lets update the main branch

**Review by matt-sasaki — APPROVED** [review]
Now it really looks good.

**Pull request merged**